SWIPs/swip-honey_to_money.md

```diff
@@ -9,29 +9,29 @@ Created: 31-07-2019
+Nodes keep track of the balances with each other nodes using an internal reference unit: honey. Since honey is not a currency in which nodes can settle their balances with each other - it is merely a unit of account - there is a requirement to facilitate converting honey to a currency. Furthermore, it is desirable that this conversion rate can fluctuate since:
 
 * The Swarm developers plan to initially set the price of honey very low and progressively increase this price to the point where it is sufficiently high to incentivize nodes to offer their service to the network while being low enough not to scare away potential users of Swarm. 
 * We expect the price of the counterpart currency to fluctuate in value against the costs of operating a Swarm node. As the real benefits (i.e. the worth of your compensation) of operating a Swarm node shouldn't fluctuate, the honey price should fluctuate inversely against the fluctuations of currency. If this happens, the real benefit stays equal, regardless of price fluctuations of the currency in which compensation is paid. 
 * The costs of operating a Swarm node is expected to decrease over time, as hardware becomes cheaper, the code more optimized etc. The price of honey is therefore expected to decrease to stay competitive to other storage solutions. 
 
-This update process should be atomic: either all nodes upgrade or none. If we would update the prices in the Swarm source code (as is currently the case), it will affect that, while not all nodes are on the same release, nodes with By externalising the price to an oracle, we can completely decouple the basic functioning of Swarm - which is heavily focused on data availability, delivery, and security considerations - from the price discovery. By building support for price oracles, we can coordinate price changes across the network, and set the stage for future experimentation with various mechanisms for setting prices, from DAOs, to bonding curves, to fixed prices. This SWIP makes no recommendation for the eventual pricing mechanism, it only prepares the network for adopting them in future.
+This update process should be atomic: either all nodes upgrade or none. If we would update the prices by releasing a new version of the Swarm source code (as is currently the case), this would be violated, as long as not all nodes are on the same release. By externalising the price to an oracle, we can completely decouple Swarm's core functions from price discovery. By building support for price oracles, we can coordinate price changes across the network, and set the stage for future experimentation with various mechanisms for setting prices, from DAOs, to bonding curves, to fixed prices. This SWIP makes no recommendation for the eventual pricing mechanism, it only prepares the network for adopting them in future.
```

Comment:
If it does not make any recommendation for the future, it should at least make clear how it will be done at the time of the implementation of this SWIP no? Is that clear how it would be?

Reply:
Valid comment. We mention elsewhere (line 20).
> The price-oracle will be initially managed by Swarm developers and stakeholders.

Which means that there is no mechanism to update prices, but the consent of Swarm-developers/stakeholders. I deliberately left-out the actual mechanism as it is an implementation detail for the price oracle. If you feel this should be included, I am very happy to include such a sentence.

Reply:
I don't think this is an implementation detail. In fact, it is absolutely not. It is an economic criteria which has nothing to do with code and which determines how incentives start working in its first iteration. Thus I do believe it should be included here.

Let's be clear with our team members who have to review this and don't have shared all the thinking we are doing internally. Let's be clear how things will work in the first iteration, this is a very general comment and applies to all similar comments.